examples/app.py

```diff
@@ -0,0 +1,117 @@
+"""A test app for pyramid-googleauth."""
+from os import environ
+from typing import List, NamedTuple
+
+from pyramid.config import Configurator
+from pyramid.response import Response
+from pyramid.security import Allowed, Denied
+from pyramid.session import SignedCookieSessionFactory
+from pyramid.view import view_config
+
+from pyramid_googleauth.security import GoogleSecurityPolicy
```

Comment:
Realised this import can be simpler:

```suggestion
from pyramid_googleauth import GoogleSecurityPolicy
```